src/python-common/ceph/deployment/service_spec.py

```diff
@@ -149,7 +149,8 @@ def __init__(self,
                  label=None,  # type: Optional[str]
                  hosts=None,  # type: Union[List[str],List[HostPlacementSpec]]
                  count=None,  # type: Optional[int]
-                 host_pattern=None  # type: Optional[str]
+                 host_pattern=None,  # type: Optional[str]
+                 default=None  # type: Optional[bool]
```

Comment:
can we keep this internal to cephadm? exposing this to the public is a bit strange.